metrics/fakes/fake_varzreporter.go

```diff
@@ -180,5 +180,4 @@ func (fake *FakeVarzReporter) recordInvocation(key string, args []interface{}) {
 	fake.invocations[key] = append(fake.invocations[key], args)
 }
 
-//lint:ignore SA1019 - auto-generated fake will go away when Varz goes away
```

Comment:
this needs to come back, this PR fails on static check because of this change. 

```
root@2a41ceeb0ae8:/repo# ./scripts/docker/test.bash
Verifying: verify_go repo/src/code.cloudfoundry.org/gorouter
go version go1.24.1 linux/arm64
Verifying: verify_go_version_match_bosh_release repo
Verifying: verify_gofmt repo/src/code.cloudfoundry.org/gorouter
Verifying: verify_govet repo/src/code.cloudfoundry.org/gorouter
Verifying: verify_staticcheck repo/src/code.cloudfoundry.org/gorouter
metrics/fakes/fake_varzreporter.go:183:7: metrics.VarzReporter is deprecated: this interface is marked for removal. It should be removed upon removal of Varz  (SA1019)
```